doc/source/quickstart.rst

```diff
@@ -93,11 +93,34 @@ named ``tools``, and you will need to run extra commands to install
 the SDK packages needed. 
 
 For Android NDK, note that modern releases will only work on a 64-bit
-operating system. If you are using a 32-bit distribution (or hardware),
-the latest useable NDK version is r10e, which can be downloaded here:
+operating system. The minimal, and recommended, NDK version to use is r17c:
```

Comment:
Well, I have done all the tests using 17c, when I first test it with travis, it failed, I don't remember why, I thing that it was related with the openssl libs (if you are interested I could search some travis build with the error). The thing is that upgrading to ndk17c solve the issue. Also, if I remember well, the unified headers feature was introduced in version 15, so all the previous versions will not work because we use that feature in our builds...maybe that version will work,  but if ndk16 fails I assume that ndk 15 will also fail. Moreover, we have to consider the stl libs, c++ is the new android default, starting from ndk17 and it is the only one in version 18, so the ndkr17 has the two lib, is prepared to successfully build with c++ (I suspect that the previous version were still immature) and is the most modern ndk  that we can use which still have the gnustl lib..so I think that is the best version to use to make the transition to the new standard...and moreover,  this version it is proved to work, so...better that we stick to the version which hast not troubles right?